mne_bids_pipeline/_download.py

```diff
@@ -43,16 +43,18 @@ def _download_from_web(*, ds_name: str, ds_path: Path) -> None:
 
     ds_path.mkdir(parents=True, exist_ok=True)
     path = ds_path.parent.resolve(strict=True)
-    fname = f"{ds_name}.zip"
+    ext = "tar.gz" if options.get("processor") == "untar" else "zip"
+    processor = pooch.Untar if options.get("processor") == "untar" else pooch.Unzip
```

Comment:
this sets up a problem that is fixed by https://github.com/fatiando/pooch/pull/458, but I wonder if it's better to set up a warning filter here rather than pin to a pooch fork / wait for pooch PR merge or even pooch release... WDYT?

Reply:
In any case we should add a comment to document why we need this differentiation :)

Reply:
Sure I could add a comment. I thought it would be evident from the names "Untar" and "Unzip" that the purpose was supporting different compression formats